src/Donations/Micros/DonationTypes/BouquetDonations.tsx

```diff
@@ -98,11 +98,7 @@ function FundingDonations({ setopenCurrencyModal }: Props): ReactElement {
                 </div> */}
                 <div className="funding-selection-option-text">
                   <span style={{ fontSize: "20px" }}>
```

Comment:
And to be able to show larger amount, just use a smaller font size, e.g. 16px?

Reply:
there can be a situation where in one screen, 6 donations are there two having biggere amounts and rest having smaller. So I don't think it wil llook nice.